src/ast/ddl.rs

```diff
@@ -228,6 +229,7 @@ pub enum TableConstraint {
         columns: Vec<Ident>,
         /// Whether this is a `PRIMARY KEY` or just a `UNIQUE` constraint
         is_primary: bool,
+        is_mysql_unique_key: bool,
```

Comment:
Please add comments to this field explaining what it means (aka how is this different)